### DISCUSSION TITLE
What is your favorite food?

### DISCUSSION THREAD

**Discussion by Hibiki-lab** [NAIST-SE/AttractingContributors#1096] [General]
https://github.com/Hibiki-lab/AttractingContributors

**Comment by hy20191108** [comment]
banaba

**Comment by Hibiki-lab** [comment]
Thanks for all comments!
We need to pull&request for completing work.
So, I would be happy if you could fill in the "List of dishes".
Best regards :)

**Comment by KazumaYamasaki** [comment]
I added to `pear` to your README.md
I would appreciate it if you could also make a pull request to [my repository](https://github.com/KazumaYamasaki/AttractingContributors-Keyboard) !
↓
https://github.com/KazumaYamasaki/AttractingContributors-Keyboard

**Comment by Ngochuy2137** [comment]
I have just contributed to your repo. Can you contribute to my repo please https://github.com/Ngochuy2137/AttractingContributors-fruits/tree/main ?

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
